acceptance/config_test.go

```diff
@@ -27,6 +27,9 @@ type TestConfig struct {
 	// If true, do not run this test against cloud environment
 	LocalOnly bool
 
+	// if true, save file repls.json with all the replacemnts
+	SaveRepls bool
```

Comment:
Could we instead of saving and then loading replacements just do the output replacement always (maybe to a separate temp folder) and then compare it? Or just always save it and ignore?

Reply:
I'd rather avoid adding this overhead to all tests.

Reply:
+1, File i/o is cheap so this should not add much overhead? Doing replacement on all output files by default makes sense since we do that anyways when performing comparisons. WDYT?

Reply:
I benchmarked it, there is no detectable difference on my laptop, so I removed the option: https://github.com/databricks/cli/pull/2352/commits/c8b4196fbcb20fa19202737f7bda3b6de4258297

We have open PRs with 100s of tests https://github.com/databricks/cli/pull/2260
so we might reconsider it then if we measure that it makes a difference.

Reply:
> And we could use diff directly instead of relying on the python script to perform replacements.

I don't see how that is possible, the replacements are applied by test runner but on file system the files are without replacements so diff will see the actual values.

Reply:
> the files are without replacements so diff will see the actual values

I was proposing we change this. If the files on the file system are with the replacement then we could use diff directly.

But on thinking about this more it would make debugging harder since since you don't know the pre-replacement values. The python approach makes sense.

Reply:
@shreyas-goenka I don't think this is possible as the replacements could be done only after `script` is finished and `diff` call is a part of the script

Reply:
Alternatively we could introduce some sort of post `script` run validation script which would call the `diff` and compare expected output but it might be more complex